media-libs/libffado/libffado-2.3.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+[[ "${PV}" = "9999" ]] && inherit subversion
+inherit eutils python-single-r1 scons-utils toolchain-funcs udev multilib-minimal
+
+DESCRIPTION="Driver for IEEE1394 (Firewire) audio interfaces"
+HOMEPAGE="http://www.ffado.org"
+
+RESTRICT="mirror"
+if [ "${PV}" = "9999" ]; then
+	ESVN_REPO_URI="http://subversion.ffado.org/ffado/trunk/${PN}"
+	SRC_URI=""
+	KEYWORDS=""
+else
+	SRC_URI="http://www.ffado.org/files/${P}.tgz"
+	KEYWORDS="~amd64 ~ppc ~x86"
```

Comment:
I actually *do* have a PPC machine with firewire running Linux, though that's coincidental and I had no intention of actually running JACK on it (nor does it run Gentoo) :)

In general, do we want to support x86? I don't care personally, but other people might. For an overlay it seems reasonable to throw in ~x86; worst case something breaks and someone complains, best case we do nothing and it just works. Stuff breaking on x86 and not amd64 is rare.